partcad/src/partcad/project_config.py

```
@@ -24,7 +24,13 @@
 class Configuration:
     name: str
 
-    def __init__(self, name, config_path=DEFAULT_CONFIG_FILENAME, include_paths=[]):
+    def __init__(
```

Ruff (0.8.2)

`__init__` is too complex (16 > 10)

(C901)